mobile/lib/models/ml/face/person.dart

```diff
@@ -6,19 +6,23 @@ import "package:flutter/foundation.dart";
 class PersonEntity {
   final String remoteID;
   final PersonData data;
+  final DateTime updatedAt;
```

Comment:
We should not need to add this property. Based on remoteID, you can get updatedAt